ci-operator/config/openshift/release/openshift-release-master__ci-4.14.yaml

```diff
@@ -421,6 +421,27 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn-crun
+- as: e2e-agent-compact-ipv4
+  cluster: build05
+  cron: 0 0 * * 1
```

Comment:
The frequency should be (at least) every 6h/12h initally as suggested in https://docs.ci.openshift.org/docs/architecture/release-gating/#add-a-periodic-job